content/finalist/ttjc@Saddamshah/index.md

```diff
@@ -0,0 +1,95 @@
+description: "expert at Javascript | Node.js | MongoDB | Currently learning React"
+---
+
+A Self-taught Web Developer was withholding more than one year of experience. Also worked for a startup company for nine months currently doing freelancing.
+
+### introduction
+
+I have more than one year of experience in the field of programming and i am very passionate about it..  I am looking forward to becoming the best version myself. 
+
+I have tried a lot of thing in my life  I was working when I was at 18 after working  3 years in BPO (call-centre) finally it been 2 years that I got to know that Codding is the skill that I love to work on. 
+
+## why are you taking this challenge?
+
+Because I want to learn with experts and enhance my codding skills 
+
+## something interesting about you?
+
+If I decide to do something So No matter how hard it is I will always find a way to complete it. One of the example is I got my first codding job while working in a BPO (call-centre) and without having a college degree. I was a commerce student before getting into codding.
+
+### help I need?
+
+I am very enthusiastic towards learning so need someone to tell me what should I need to learn basically a mentor.
+I have a MEN Stack  (MongoDB Express Node). Currently working on react.js.
+
+### how to contact me?
+
+- [Outlook](mailto:shah.saddam@outlook.com)
+- [LinkedIn](https://www.linkedin.com/in/saddam-shah/)
+
+### Projects
+
+## [ Project No 1 - E Commerce Store ]
+_description_ A full fledged E-commerce Web Application which has 13 unique pages including two dashboards and one payment gateway Where users can view all the products that are differentiated by their categories. 
+Backend is created on Node server and frontend is built on react.js
+Full Authentication system using JWT, where each signup data is stored on MongoDB database using mongo atlas.
+
+_hosted link_ https://fashionstoreproject.netlify.app
+_github link_ https://github.com/Saddamshah/fashion-store
+_API github link_ https://github.com/Saddamshah/fashionstoreapi
+
+Kindly read the depth overview in the GitHub readme file on this repository.
+
+## [ Project No 2 Coffee Shop ] 
+_description_ A web template for locals business created using react.
+
+_hosted link_ https://project-coffee-shop.netlify.app/
+_github link_ https://github.com/Saddamshah/coffee-shop
+
+
+##  [ Project No 3 Book Search App ] 
+_description_ A simple book search app using react
+
+_hosted link_ https://book-search-project.netlify.app/
+_github link_ https://github.com/Saddamshah/book-search
+
+
+##  [ Project No 4 Data Visualization Map ]
+_description_ A Data Visualization that show covid cases on map.
+
+_hosted link_ https://data-visualization-covid.herokuapp.com/
+_github link_ https://github.com/Saddamshah/Data-Visualization-covid-
+
+##  [ Project No 5 Currency Converter ]
+_description_ A Currency Converter app using react.
+
+ _hosted link_ https://app.netlify.com/sites/currency-converter-project
```

Comment:
You have not provided the correct hosted link for the project kindly check them.